content/2017/02/09/generating-documentation-polyglot-projects.rst

```diff
@@ -0,0 +1,123 @@
+public: yes
+author: david
+tags: [
+  Docs,
+  Tools,
+  Javascript,
+  Python,
+  Sass
+  ]
+summary: |
+  Code documentation is ideally written as close to the actual code
+  as possible, but compiled into a comprehensive set of documentation
+  that includes code from all languages in use. Here's how we intend
+  to do that.
+
+
+Generating code documentation for polyglot projects
```

Comment:
Can we update this to Title Case? :) That will follow the new guidelines that we posted while you were gone and probably didn't tell you about.